Src/ILGPU.Tests.SPIRV/SPIRVBuilderUtilsTests.cs

```diff
@@ -0,0 +1,64 @@
+using ILGPU.Backends.SPIRV;
+using System.Collections.Generic;
+using Xunit;
+
+namespace ILGPU.Tests.SPIRV
+{
+    public class SPIRVBuilderUtilsTests
+    {
+        [Fact]
+        public void ToUintListUint()
+        {
+            var result = SPIRVBuilderUtils.ToUintList((uint)0);
```

Comment:
General question: should we abstract from the actual `uint` type by introducing a new `SpirVInstruction` structure? This would make the code easier to understand and all types will be `CLSCompliant`.

Reply:
What exactly would you want to include in this structure? Basically the only thing that all ops share is an OpCode and WordCount. Perhaps I'm misunderstanding...